charts/kube-prometheus-stack/Chart.yaml

```
@@ -21,7 +21,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 52.0.1
+version: 52.1.1
```

Sorry for the confusion, I just suggested the minor change on the previous, but this should be what we want, since we're adding a feature:
```suggestion
version: 52.1.0
```